dbscripts/xml/install.xml

```diff
@@ -23,6 +23,7 @@
 	<migration class="lib.pkp.classes.migration.FilesMigration" />
 	<migration class="lib.pkp.classes.migration.GenresMigration" />
 	<migration class="lib.pkp.classes.migration.JobsMigration" />
+	<migration class="lib.pkp.classes.migration.FailedJobsMigration" />
```

Comment:
With this, if a job failed, he will be persisted into the failed_jobs table. Helping us to tracking down failed jobs.